src/app/community/_components/AuthorCard.tsx

```diff
@@ -12,39 +13,71 @@ interface AuthorCardProps {
   nickname: string;
   dateText: string;
   userImage: string | null;
+  userId: number;
   onClickPopOver: () => void;
   onClosePopOver: () => void;
   isOpenPopOver: boolean;
   popOverOptions: {
     label: string;
-    onClick: (e: React.MouseEvent<HTMLDivElement>) => void;
+    onClick?: () => void;
   }[];
 }
 
 export default function AuthorCard({
   nickname,
   dateText,
   userImage,
+  userId,
   onClickPopOver,
   onClosePopOver,
   isOpenPopOver,
   popOverOptions,
 }: AuthorCardProps) {
+  const userProfileCardRef = useRef<HTMLDivElement>(null);
+  const [isOpenUserCard, setIsOpenUserCard] = useState(false);
+  const [isHovering, setIsHovering] = useState(false);
+
+  const handleOpenProfile = () => {
+    setIsOpenUserCard(true);
+  };
+
+  const handleCloseProfile = () => {
+    setIsOpenUserCard(false);
+  };
+
   return (
-    <div className={cn('container')}>
-      <ProfileImage profileImage={userImage} />
+    <div
+      className={cn('container')}
+      onClick={(e) => e.stopPropagation()}
+      onMouseEnter={() => setIsHovering(true)}
+      onMouseLeave={() => setIsHovering(false)}
+    >
+      <div
+        onMouseEnter={handleOpenProfile}
+        onMouseLeave={handleCloseProfile}
+        className={cn('user-profile')}
+        ref={userProfileCardRef}
+      >
```

Comment:
```js
  const profileHandlers = {
    onMouseEnter: handleOpenProfile,
    onMouseLeave: handleCloseProfile,
  };
  ```
  함수를 이런식으로 만들고 
```suggestion
      <div
        className={cn('user-profile')}
        ref={userProfileCardRef}
        {...profileHandlers}
      >
```
스프레드 문법으로 내려주는 방식도 있ㅅ브니다

Reply:
@bokeeeey 오호 좋은데요? 츄베릅 하겠슴다